results/RESULTS.md

```diff
@@ -0,0 +1,10 @@
+# Performance and Scale Results
+gatling-puppet-load-test results are now copied to this directory rather than the parent directory.
```

Comment:
should "this" be "these"?  referring to the below sections...

Reply:
Updated